### ISSUE TITLE
About the training scripts.

### ISSUE THREAD

**Issue by Zjshadow** [Edward-Sun/easy-to-hard#3]
I have two questions about the script that trains the reward model:

1. In the example, the entry file of the training reward model is finetune_rm.py. In the actual project code, does it refer to train_rm_pointwise.py or train_rm_pairwise.py?
2. train_rm_pointwise.py and train_rm_pairwise.py look exactly the same, so what is the difference between the training purpose of these two files? If the purpose is different, how does the same document reflect the difference between the two training modes?

Looking forward to the answer. Thanks.

**Comment by Edward-Sun** [comment]
Hi @Zjshadow 

1) For math problems, typically we only have 0-1 rewards for solutions, so we used `train_rm_pointwise.py`. I'll update the example to reflect this.

2) Good catch... It seems I uploaded the wrong file for `train_rm_pairwise.py`. It has been fixed.

Thanks,
Zhiqing
